### PR TITLE
BigQuery: Support multiple projects.

### PR DESCRIPTION
**Pull request by ilya-lt**
Support multiple projects in the schema fetch process.

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source)
- [ ] New Alert Destination
- [ ] Other

### CODE REVIEW

File: redash/query_runner/big_query.py [hunk 1]
```diff
@@ -101,6 +101,10 @@ def configuration_schema(cls):
                     'type': 'string',
                     'title': 'Project ID'
                 },
+                'additionalProjects': {
```
Comment (dgilboa) [inline redash/query_runner/big_query.py:104]: the change here is only for the big query backend service - what about the data source config update page (separate PR?)

**Review by dgilboa — CHANGES_REQUESTED** [review]
GG !

what about updating the date source config ?

File: redash/query_runner/big_query.py (hunk 1 above)
Comment (ilya-lt) [reply redash/query_runner/big_query.py:104]: There isn't a datasource config page for big query, or any other datasource, that's all the config available, you can't validate the config, change the html controls, or anything of this sort in the current way redash is designed, only add and remove fields of primitive types.

**Review by dgilboa — CHANGES_REQUESTED** [review]
the actual feature changes looks ok and can be approved.
please extract the version set to a different PR (or just release tag as done in this repo) before actual approve to this PR.

**Review by dgilboa — APPROVED** [review]
LGTM

**Pull request merged**